spel/minimal-linux.json

```diff
@@ -459,6 +516,12 @@
         "azure_vm_size": "Standard_DS2_v2",
         "iso_url_centos7": "http://mirror.cs.vt.edu/pub/CentOS/7/isos/x86_64/CentOS-7-x86_64-Minimal-2009.iso",
         "iso_url_centos8": "http://mirror.cs.vt.edu/pub/CentOS/8/isos/x86_64/CentOS-8.1.1911-x86_64-dvd1.iso",
+        "openstack_allow_insecure_connection": "false",
+        "openstack_flavor_name": "",
+        "openstack_floating_ip_network_name": "",
+        "openstack_network_id": "",
+        "openstack_security_group_name": "",
```

Comment:
```suggestion
        "openstack_security_group_names": "",
```

Reply:
Done (https://github.com/plus3it/spel/pull/439/commits/e7b7642a993ecbd49b11834846e40f0d57d333c0).  Tested it, and it still works with single-item lists (i.e., no commas).  Good call!